packages/wallets/src/hooks/useInputATMFormatter.ts

```diff
@@ -21,10 +21,8 @@ const useInputATMFormatter = (initial?: number, options?: TOptions) => {
         (e: DeepPartial<React.ChangeEvent<HTMLInputElement>> | React.ChangeEvent<HTMLInputElement>) => {
             const newValue = e?.target?.value || '';
             const unformatted = unFormatLocaleString(newValue, locale);
-            // @ts-expect-error shouldn't cast to number because we will lose the trailing zeros.
-            const shifted = Math.fround(unformatted * 10).toFixed(fractionDigits);
-            // @ts-expect-error shouldn't cast to number because we will lose the trailing zeros.
-            const unShifted = Math.fround(unformatted / 10).toFixed(fractionDigits);
```

Comment:
Doing `Math.fround` here was decreasing the precision by forcing the result into 32 bit instead of 64

<img width="289" alt="Screenshot 2023-11-15 at 18 08 30" src="https://github.com/binary-com/deriv-app/assets/119863957/96760a12-afa7-4cb7-9bbc-8fc0faf7978c">

Reply:
hmm, will decreasing precision actually help? It does decrease precision in terms of size of bits (digital representation), but in terms of decimal point representation/strings, it seems to be generating bigger floating point calculation errors and actually bigger risk of those errors being visible.

Reply:
Historially, the only truly reliable approach I've found is to use lodash _.round function, which is internally based on scientific notation / string operations (if rounding is the goal).

https://github.com/lodash/lodash/blob/main/src/.internal/createRound.ts

Reply:
pretty cool to see that improved from .fround :D good catch :)